src/button/button.js

```diff
@@ -64,9 +64,9 @@ class Button extends React.Component<
       ...restProps
     } = this.props;
     // Get overrides
-    const [BaseButton, baseButtonProps] = getOverrides(
-      overrides.BaseButton,
-      StyledBaseButton,
+    const [Root, RootProps] = getOverrides(
+      overrides.Root || overrides.Root,
```

Comment:
```suggestion
      overrides.Root || overrides.BaseButton,
```

Reply:
yeah, I was too eager replacing stuff :D  fixing all these now

Reply:
I am having seconds thoughts on if we should change the name in JSX from BaseButton to Root - probably folks would have to update jest snapshots tests

maybe not a big deal, but by not changing it, it would be less of a headache - what do you all think?